pdr_backend/cli/cli_arguments.py

```diff
@@ -47,11 +47,12 @@
   pdr multisim PPSS_FILE
   pdr arima_plots PPSS_FILE [--debug_mode False]
   pdr deployer (for >1 predictoor bots)
+  pdr ohlcv PPSS_FILE NETWORK
   pdr lake raw|etl update PPSS_FILE NETWORK
   pdr lake raw|etl drop PPSS_FILE NETWORK ST
   pdr lake describe --HTML PPSS_FILE NETWORK
   pdr lake validate PPSS_FILE NETWORK
-  pdr analytics PPSS_FILE NETWORK
+  pdr predictoor_dashboard PPSS_FILE NETWORK
```

Comment:
this should be called `pdr dashboard PPSS_FILE NETWORK`

Reply:
I see latest changes has been merged from main but this command still didn't change